modules/ROOT/pages/oauth2-client-store.adoc

```diff
@@ -0,0 +1,75 @@
+= Mule OAuth 2.0 Client Store Configuration
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Mule OAuth 2.0 Client Store allows the Mule OAuth 2.0 Provider to use Anypoint Client Applications for token generation.
+
+This feature is available for Mule 4.2.0 and later.
+
+== Prerequisites
+
+The Mule runtime engine hosting the Mule OAuth 2.0 Provider must meet two requirements:
+
+* It must be licensed to run an API Gateway. +
+See xref:mule-runtime::installing-an-enterprise-license.adoc[Install Enterprise License] for more information.
+* It must have configured Anypoint platform credentials. +
+ Follow the xref:org-credentials-config-mule4.adoc[Configuring Organization Credentials in Mule Runtime 4] for more information.
+
+== OAuth 2.0 Client Store Properties
+
+The Object Store has two configurable properties:
+[%header%autowidth.spread,cols="a,a"]
+|===
+| `anypoint.oauth_provider.cache_persistence_enabled`
+| Defines whether the caching mechanism should persist locally. The default value is `true` for when the property is not set or cannot be parsed. Cache persistence allows client authorization without connection to Anypoint Platform.
+
+| `anypoint.oauth_provider.cache_ttl_days`
+| Represents how many days each Client Application should be cached by the OAuth Server. This value can be any positive floating point number. The default configuration is `30`, representing 30 days.
+|===
+
+== Usage
+
+To set the Mule OAuth Client Store in your OAuth Provider, use the following Object Store in the 'clientStore' parameter of the oauth-provider:config element.
```

Comment:
'clientStore' > these are meant to be backticks? 
oauth-provider:config > Should this be inside backticks also?